starboard/player.h

```diff
@@ -614,6 +614,12 @@ SB_EXPORT bool SbPlayerSetPlaybackRate(SbPlayer player, double playback_rate);
 //   value of |1.0| means that it should be played at full volume.
 SB_EXPORT void SbPlayerSetVolume(SbPlayer player, double volume);
 
+// Sets maximum size of a buffer of input data.
+//
+// |max_input_size|: The maximum size of a buffer of input data.
+//   Use |-1| for default value.
+SB_EXPORT bool SbPlayerSetMaxInputSize(int max_input_size);
```

Comment:
We shouldn't add a new Starboard function for this feature.  In general, even if we are going to add a new function, the function should be guarded by Starboard version.

For this particular feature, we should use a Starboard extension.

Reply:
Moved to Starboard extension.

Reply:
As we need to set `max_input_size` per stream, moved this to HTMLVideoElement instead of Starboard extension.

Reply:
Discussed with Jason offline, will change this to Starboard extension with a global map to store (player_pointer, max_input_size) for this feature.